docs/2023-06-08-meeting-transcript.md

```diff
@@ -0,0 +1,881 @@
+
+Um, so the first question was really which tests are now happening. I want to talk about this from a faily high-level perspective, and the reason why I don't be extremely detailed is we don't want to be in a place, where, everything is out there and open, and people understand exactly which test are performed, and how and maybe we see the code, that is running those tests, so they can find ways to bypass that, so. 
+
+I understand that ambiguity is, um, a concern, especially, and this is why I wanna try sharing as much as I can, from the perspective how we work, but at the same time, hopefully some reviewers can understand that, exposing everything that eventually would lead to vulnerabilities. 
+
+So, for every modded submission, for every submission of the mods, we run automatic checks and then we run, for some mods, manual checks. Manual checks are not being triggered for every single mod update. For some, we're just happy with the automation checks we have.   
+
+Technically, we start by running antivirus scans and malware scans, and uh we use couple of services, um, to do that. 
+
+We check for file tags and inspecting file structures, um, I saw the comment in the kinda of the background of this call on um, signing, antiviruses and SHA-1, just emphasises that our API actually supports um SHA-1, so theroretically we could've used it. There is, um, ability to do that. So we're taking this into account; we are making more clients improvements to make sure this is enforced in a good way.  
+
+Um, in terms of the manual checks, whenever something is triggered, um from these automation checks, then a human would then review the mod. By the way, if someone changes the texts and images, we also require manual review, to avoid profanity and that content, all of these things, um.  
+
+But if those automated checks have trigged some sort of the issue, or there is suspicous, it would be an manual check. 
+
+Also I wanna emphasis that this is continuously evolving process for us, uh so just for example following this incident, we already updated our automatic checks, to um test for this particular vulnerabiliy, and should we identified this particular vulnerability again, it would trigger a manual review, um because we want to double check to make sure that, um, the mod are not compromised.    
+
+I also saw the idea of um [DISCONNECTED]
+
+Emi: ah, looks like they've muted. We also have the Geometrically from Modrinth who's here to speak, so uh, if you have any finishing words, go ahead. We want other voices. 
+
+Fury: sorry, uh I got disconnected I think. Can you guys hear now?
+ 
+Emi: yes.
+
+Fury: OK. Can you repeat? I did not listen; I am not here for the first thing what's that?
+
+Emi: Since you muted, and because we want to get as many voices in here as possible, we're going to [be] moving on to modern's perspective, if you have any quick final words, feel free.
```

Comment:
```suggestion
Emi: Since you muted, and because we want to get as many voices in here as possible, we're going to [be] moving on to Modrinth's perspective, if you have any quick final words, feel free.
```

Reply:
*faceplam moment*